js/mobileMenu.js

```diff
@@ -0,0 +1,7 @@
+      $(document).ready(function() {
+      $(document, '#js-bootstrap-offcanvas').on('click, touch', function(){ 
```

Comment:
Indentation is wrong, move this by two spaces, check Javascript indentation convention